sites/footers/includes/footer.html

```diff
@@ -1,29 +1,86 @@
+				{%- for link in page.contextualFooter.links -%}
+				<li><a href="{{ link.url }}">{{ link.text }}</a></li>
+				{%- endfor -%}
+				{%- else -%}
+				{%- for link in site.global.contextualFooter.links[i18nText-lang] -%}
+				<li><a href="{{ link.url }}">{{ link.text }}</a></li>
+				{%- endfor -%}
+				{%- endif -%}
+			</ul>
+		</nav>
+	</div>
+{%- endif -%}
+{% endunless %}
+
 {% unless page.noFooterAbout %}
 	<div class="landscape">
-		<nav class="container wb-navcurr">
-			<h2 class="wb-inv">{{ i18nText-footerAbout }}</h2>
-			<ul class="list-unstyled colcount-xs-2 colcount-md-3">
-{%- if i18nText-lang == "fr" -%}
-				<li><a href="https://www.canada.ca/fr/contact.html">Contactez-nous</a></li>
+		<nav class="container">
+			<h3>{{ i18nText-goc }}</h3>
+			<ul class="list-col-xs-1 list-col-sm-2 list-col-md-3">
+				{%- if i18nText-lang == "fr" -%}
+				<li><a href="https://www.canada.ca/fr/contact.html">Toutes les coordonnées</a></li>
 				<li><a href="https://www.canada.ca/fr/gouvernement/min.html">Ministères et organismes</a></li>
-				<li><a href="https://www.canada.ca/fr/gouvernement/fonctionpublique.html">Fonction publique et force militaire</a></li>
-				<li><a href="https://www.canada.ca/fr/nouvelles.html">Nouvelles</a></li>
-				<li><a href="https://www.canada.ca/fr/gouvernement/systeme/lois.html">Traités, lois et règlements</a></li>
-				<li><a href="https://www.canada.ca/fr/transparence/rapports.html">Rapports à l'échelle du gouvernement</a></li>
-				<li><a href="https://pm.gc.ca/fra">Premier ministre</a></li>
-				<li><a href="https://www.canada.ca/fr/gouvernement/systeme.html">Comment le gouvernement fonctionne</a></li>
-				<li><a href="https://ouvert.canada.ca/">Gouvernement ouvert</a></li>
-{%- elsif i18nText-lang == "en" -%}
-				<li><a href="https://www.canada.ca/en/contact.html">Contact us</a></li>
+				<li><a href="https://www.canada.ca/fr/gouvernement/systeme.html">À propos du gouvernement</a></li>
+				{%- elsif i18nText-lang == "en" -%}
+				<li><a href="https://www.canada.ca/en/contact.html">All Contacts</a></li>
 				<li><a href="https://www.canada.ca/en/government/dept.html">Departments and agencies</a></li>
-				<li><a href="https://www.canada.ca/en/government/publicservice.html">Public service and military</a></li>
-				<li><a href="https://www.canada.ca/en/news.html">News</a></li>
-				<li><a href="https://www.canada.ca/en/government/system/laws.html">Treaties, laws and regulations</a></li>
-				<li><a href="https://www.canada.ca/en/transparency/reporting.html">Government-wide reporting</a></li>
-				<li><a href="https://pm.gc.ca/eng">Prime Minister</a></li>
-				<li><a href="https://www.canada.ca/en/government/system.html">How government works</a></li>
-				<li><a href="https://open.canada.ca/en/">Open government</a></li>
+				<li><a href="https://www.canada.ca/en/government/system.html">About government</a></li>
+				{%- endif -%}
+			</ul>
+			<hr />
+			<h4 class="wb-inv">{{ i18nText-themeLinks }}</h4>
```

Comment:
@duboisp Just so we're clear, this would mean a lot of CSS to disable the `.wb-inv` class on the `h4` and then some more to hide the text without impacting the white bar. I propose, if we go forward with this, to add the after peudo element to the `ul` above instead. What do you think?

CC @GormFrank